core-aam/index.html

```diff
@@ -6372,7 +6372,7 @@ <h4 id="ariaColIndexText"><code>aria-colindextext</code></h4>
               <tr>
                 <th><abbr title="macOS Accessibility Protocol">AX API</abbr></th>
                 <td>
-                  <span class="property">Property: TBD</span>
+                  <span class="property">Property: <code>AXARIAColumnIndexText</code>: <code>&lt;value&gt;</code></span>
```

Comment:
@twilco et al discussing whether we should double-down on having "ARIA" in the property name just because that was added to the related `AXARIAColumnIndex` property.... We may deprecate the ARIA-specific names in the future anyway.
```suggestion
                  <span class="property">Property: <code>AXColumnIndexText</code>: <code>&lt;value&gt;</code></span>
```